Sources/Unbox.swift

```diff
@@ -47,6 +47,13 @@ public func Unbox<T: Unboxable>(dictionary: UnboxableDictionary, at key: String,
     return container.model
 }
 
+/// Unbox a JSON dictionary into a model `T` beginning at a provided key, optionally using a contextual object. Throws `UnboxError`.
+public func Unbox<T: UnboxableWithContext>(dictionary: UnboxableDictionary, at key: String, isKeyPath: Bool = true, context: T.ContextType? = nil) throws -> T {
```

Comment:
The context can't be `nil` for `UnboxableWithContext`, so it shouldn't be optional.